generic/tests/cfg/jumbo.cfg

```diff
@@ -8,16 +8,16 @@
         mtu = 1500
     e1000:
         Linux:
-            mtu = 16110
+            mtu = 9000
```

Comment:
Now need set switch all port to the same MTU, as bridge , the Maximum value only can set 9000. you can see https://bugzilla.redhat.com/show_bug.cgi?id=1348105#c7